etna/auth0/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path("login/", views.login, name="account_login"),
+    path("logout/", views.logout, name="account_logout"),
+    path("signup/", views.register, name="account_signup"),
+    path("authorize/", views.authorize, name="account_authorize"),
+]
```

Comment:
NOTE: The intention here is to mimic `allauth` where possible in terms of names / paths, so that you can always use `django.urls.reverse()` or the `{% url 'name_of_url' %}` tag to link to the key views, regardless of whether AUTH0 is configured/enabled.